frontend/public/module/k8s/cluster-operator.ts

```diff
@@ -5,15 +5,15 @@ import { ClusterOperator, OperandVersion } from '.';
 export enum OperatorStatus {
   Available = 'Available',
   Updating = 'Updating',
-  Failing = 'Failing',
+  Degraded = 'Degraded',
   Unknown = 'Unknown',
 }
 
 export const getStatusAndMessage = (operator: ClusterOperator) => {
   const conditions = _.get(operator, 'status.conditions');
-  const failing: any = _.find(conditions, { type: 'Failing', status: 'True' });
-  if (failing) {
-    return { status: OperatorStatus.Failing, message: failing.message };
+  const degraded: any = _.find(conditions, { type: 'Degraded', status: 'True' });
```

Comment:
Not sure if we shouldn't fallback to the `Failing` type if the operator won't be updated with the `Degraded` condition type. Not sure if we can count that all the operators will be updated right away...
Together with a ToDo note...

Reply:
All operators will have to be updated before we ship, and it won't cause errors if an operator isn't updated. I'd rather code to the new API. Then we're not leaving things in that we'll just have to remove later anyway.